parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -267,22 +267,40 @@ retrycmd_nslookup() {
     current_time=$(date +%s)
     echo "Executed nslookup -timeout=$timeout -retry=0 $record for $((current_time - start_time)) seconds";
 }
-retrycmd_get_tarball() {
-    tar_retries=$1; wait_sleep=$2; tarball=$3; url=$4
-    echo "${tar_retries} retries"
-    for i in $(seq 1 $tar_retries); do
-        [ -f "$tarball" ] && tar -tzf "$tarball" && break || \
-        if [ "$i" -eq "$tar_retries" ]; then
+
+_retry_file_curl_internal() {
+    retries=$1; waitSleep=$2; timeout=$3; filePath=$4; url=$5; checksToRun=( "${@:6}" )
```

Comment:
non-blocking: did we decide to move towards camelCase? I'm fine with it I just think we should be consistent and not mix with snake_case if possible

Reply:
We did not, it is kind of all over the place right now, we could maybe agree on one style and refactor? Because i've seen both versions being used interchangeably